idea/tests/org/jetbrains/kotlin/idea/debugger/sequence/psi/java/LocationPositiveChainTest.kt

```diff
@@ -0,0 +1,41 @@
+// Copyright 2000-2017 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+package org.jetbrains.kotlin.idea.debugger.sequence.psi.java
+
+/**
+ * @author Vitaliy.Bibaev
+ */
+class LocationPositiveChainTest : PositiveJavaStreamTest("location") {
```

Comment:
Why can't we generate this test (and also others) from the test data?

Reply:
Yes, we can. Will be fixed later.